src/components/popup/popup.ts

```diff
@@ -68,6 +68,10 @@ export class MPopup extends ModulVue {
     @Prop()
     public trigger: HTMLElement;
 
+    public $refs: {
```

Comment:
je ne vois pas trop le besoin d'ajouter une ref sur popper?

Reply:
Pour avoir une référence à la composante popper dans la liste déroulante. Est-ce qu'il y a un autre moyen?
this.$refs.popup.$refs.popper.update();

Reply:
Ça m'a échappé! En fait, il ne faudrait pas exposer le $ref du $ref. Il faudrait plutôt que dans le dropdown on appelle `this.$refs.popup.update()`, et dans le popup on ait un update qui lui fait `this.$ref.popper.update()`